process/generators.py

```diff
@@ -54,13 +60,13 @@ def nv_all_units(proj):
     # TODO: more useful when we have more than just one data source
     result = []
     if proj.field('market_rate_units_net'):
-        result.append(NameValue('net_num_units',
-                                proj.field('market_rate_units_net'),
-                                'planning'))
+        result.append(OutputNameValue('net_num_units',
+                                      proj.field('market_rate_units_net'),
```

Comment:
Hmm I'm trying to understand this, wouldn't this mean that when we look at the fields (and use the weighted system + business rules) it's possible it returns a PTS field but we label it planning here?

Reply:
you're right. also in practice, i think field names will rarely be identical across sources.

i'm changing project.field to require a source. lots of incoming related changes as well.